### PR TITLE
Historical Catalogue map with dev-data

### LINKED ISSUE #40: Add ISC and GHEC maps
Add map creation to be also able to run tests

### PR DESCRIPTION
**Pull request by hascar**
Inserted programmatically the Map and Tests for Historical Catalogue with dev-data
The tests are green here: https://ci.openquake.org/job/zdevel_oq-platform2/424/
Close #40

### CODE REVIEW

File: openquakeplatform/test/ghec_test.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        pla.wait_new_page(enter_button, '/maps/%s' % id_map)
+
+        enter_button = pla.xpath_finduniq(
+            "//a[@href='/maps/%s/view' and "
+            "normalize-space(text()) = 'View Map']" % id_map)
+        enter_button.click()
+        pla.wait_new_page(enter_button, '/maps/%s/view' % id_map, timeout=15)
+
+        import time
+        time.sleep(5)
```
Comment (nastasi-oq) [inline openquakeplatform/test/ghec_test.py]: sleep required ?

Comment (hascar) [reply openquakeplatform/test/ghec_test.py]: Fixed this here: https://github.com/gem/oq-platform2/pull/46/commits/1d69b5794311d8dbf5821663bc452ec4a342e714

File: openquakeplatform/ghec_viewer/management/commands/create_ghecmap.py [hunk 2]
```diff
@@ -0,0 +1,175 @@
+# Copyright (c) 2012-2013, GEM Foundation.
```
Comment (nastasi-oq) [inline openquakeplatform/ghec_viewer/management/commands/create_ghecmap.py]: Update dates.

File: openquakeplatform/utils.py [hunk 3]
```diff
@@ -58,6 +58,13 @@ def oq_context_processor(request):
     except:
         context['ISC_MAP_ID'] = 23
 
+    try:
+        ghec_map = Map.objects.filter(
+            title_en='Global Historic Catalogue')
```
Comment (nastasi-oq) [inline openquakeplatform/utils.py]: Please substitute title_en with `uuid` field (and do the same for ISC map).

**Pull request merged**